Back End/MemorizeWords/MemorizeWords/Api/ApplicationVariableApiInitializer.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace MemorizeWords.Api
+{
+    public static class ApplicationVariableApiInitializer
```

Comment:
Please move this class to Apis folder here "./Back End/MemorizeWords/MemorizeWords/Api/Apis"